src/notifyme.js

```diff
@@ -1,116 +1,117 @@
-/**
- * Global Variable NotifyMe
- */
-var NotifyMe = {};
-(function (window, NotifyMe) {
-    var Notify;
-    Notify = (function () {
-        var getPermissionAndLaunch, launchNotiWindow, mergeObjects, defaultOptions, generateGuid;
+(function(root, definition) {
+    if (typeof define === 'function' && define.amd) {
+        define([], definition);
+    } else {
+        root.NotifyMe = definition();
+    }
+})(this, function() {
+    var defaultOptions = {
+        icon: 'http://i.istockimg.com/file_thumbview_approve/46749378/3/'
```

Comment:
I updated the constructor signature to be `function NotifyMe(title, options)` to match the `Notification` constructor https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification